apps/pie-docs/src/_includes/pie-logo.njk

```diff
@@ -1 +1,26 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 60 24"><g fill="#242E30" clip-path="url(#a)"><path d="M23.972 23.94h3.327a.321.321 0 0 0 .323-.318v-5.727h.053c.78.936 2.175 1.418 3.84 1.418 3.785 0 6.55-2.996 6.55-7.062 0-4.066-2.765-6.795-6.55-6.795-1.8 0-3.277.588-4.055 1.606h-.053v-.967a.321.321 0 0 0-.323-.318h-3.112a.321.321 0 0 0-.323.318v17.527c0 .175.145.318.323.318Zm6.847-8.32c-1.987 0-3.195-1.444-3.195-3.235 0-1.792 1.208-3.236 3.195-3.236 1.986 0 3.195 1.444 3.195 3.236 0 1.79-1.209 3.236-3.195 3.236ZM44.464 2.247c0-1.203-.886-2.247-2.362-2.247-1.477 0-2.363 1.043-2.363 2.247 0 1.205.886 2.22 2.363 2.22 1.476 0 2.362-1.016 2.362-2.22Zm-4.026 16.745h3.328a.321.321 0 0 0 .322-.318V6.095a.321.321 0 0 0-.322-.318h-3.328a.321.321 0 0 0-.323.318v12.579c0 .175.145.318.323.318ZM53.136 19.313c2.287 0 4.196-.842 5.246-1.96a.315.315 0 0 0-.012-.443l-1.966-1.893a.335.335 0 0 0-.444-.021c-.611.521-1.64.838-2.69.838-1.19 0-2.403-.506-2.942-1.626a.32.32 0 0 1 .29-.46h8.59a.32.32 0 0 0 .319-.265c.058-.339.106-.79.106-1.098 0-3.236-1.88-6.929-6.713-6.929s-6.794 3.691-6.794 6.929c0 3.237 2.201 6.928 7.009 6.928h.001Zm-.214-10.54c1.284 0 2.372.587 2.746 2.019a.162.162 0 0 1-.156.202h-5.206a.162.162 0 0 1-.157-.202c.375-1.432 1.464-2.018 2.773-2.018ZM11.684 7.777a.162.162 0 0 0 .227 0l2.484-2.446a.159.159 0 0 0-.033-.251 9.792 9.792 0 0 0-2.07-.876.164.164 0 0 0-.162.041L10.229 6.12a.157.157 0 0 0 0 .224l1.456 1.434ZM16.823 14.497a.162.162 0 0 0-.227 0l-1.456 1.434a.157.157 0 0 0 0 .224l2.46 2.423a.163.163 0 0 0 .253-.028c.387-.62.705-1.286.942-1.987a.159.159 0 0 0-.04-.163l-1.933-1.905.001.002Z"/><path d="m5.201 14.38 3.474-3.421.004.003.479-.472a.162.162 0 0 1 .228 0l2.998 2.953a.157.157 0 0 1 0 .224l-2.187 2.154a.157.157 0 0 0 0 .224l1.456 1.434a.162.162 0 0 0 .227 0l6.912-6.808a.156.156 0 0 0 .04-.162 9.527 9.527 0 0 0-.926-2.003.163.163 0 0 0-.253-.028l-3.355 3.304a.163.163 0 0 1-.228 0l-7.563-7.45a.165.165 0 0 0-.167-.039 9.944 9.944 0 0 0-2.017.928.158.158 0 0 0-.029.248l3.18 3.133a.157.157 0 0 1 0 .224l-2.997 2.953a.162.162 0 0 1-.228 0L1.135 8.716a.162.162 0 0 0-.255.034 9.5 9.5 0 0 0-.873 2.056.159.159 0 0 0 .042.158l2.515 2.478a.157.157 0 0 1 0 .224L.08 16.113c-.042.042-.058.104-.042.16.218.716.517 1.4.888 2.035.053.09.179.105.255.032L5.2 14.38Z"/><path d="m10.084 17.591-3.116-3.069a.162.162 0 0 0-.227 0l-1.425 1.404a.157.157 0 0 0 0 .224l2.159 2.127a.157.157 0 0 1 0 .224l-3.097 3.05a.159.159 0 0 0 .03.25 9.854 9.854 0 0 0 2.033.912c.058.019.122.003.164-.04l2.551-2.513a.162.162 0 0 1 .228 0l2.638 2.599c.042.041.104.057.16.041a9.854 9.854 0 0 0 2.048-.838.157.157 0 0 0 .034-.25l-4.185-4.122.005.002Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h59.707v24H0z"/></clipPath></defs></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 60 24">
```

Comment:
I know it was the same before your PR, but why do we need both this file and the SVG below?

Reply:
good question, I would assume it's to render the logo as an SVG instead of img, or to avoid any extra assignments to make the import work on all pages like what I did with the jet logo [here](https://github.com/justeattakeaway/pie/pull/269/files#diff-4d23f6020146d70080f2f2d7f9344219b5c80df94d09bfaaf4f43c7a33b3d70cR6-R7), but I might be wrong, so cc @jamieomaguire :D

Reply:
previously we used `{% include "pie-logo.njk" %}` in both the header and footer. we can use a single approach now I think. would an inline svg be present immediately on page load vs loading an image in from the server? that could be a case for inlining on both the header and footer rather than an `<img>` tag

Reply:
Sorry, I misunderstood the above. It looks like we can delete the `svg` file as I can't see any references in the project to it